src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/entity/ObjectionsLinkKeys.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.companieshouse.api.strikeoffobjections.model.entity;
+
+import uk.gov.companieshouse.service.links.CoreLinkKeys;
+import uk.gov.companieshouse.service.links.LinkKey;
+
+public enum ObjectionsLinkKeys implements LinkKey {
```

Comment:
I think just ObjectionLinkKeys for this class name. Reads a bit easier and we've tended to avoid using plural Objections.

Reply:
Changed the name as suggested.